config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -1,5 +1,7 @@
 extensions:
   health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
```

Comment:
is this a valid config? I understood that our healthcheck agent was using hard coded path to "/".

Reply:
Thanks. Updated.